cvpysdk/backupset.py

```diff
@@ -494,6 +511,22 @@ def __init__(self, instance_object, backupset_name, backupset_id=None):
         self.subclients = Subclients(self)
         self.schedules = Schedules(self)
 
+        self.default_browse_options = {
+            'operation': 'browse',
+            'show_deleted': False,
+            'from_time': 0,  # Timestamp or %Y-%m-%d %H:%M:%S

```

Comment:
Timestamp provided can be either string or int.
If it is formatted time, then it should be a string